buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -221,7 +222,7 @@ public void afterExecute(Task task, TaskState state) {
                     if (state.getFailure() != null) {
                         // If the task fails, and other tasks use this cluster, the other task will likely never be
                         // executed at all, so we will never get to un-claim and terminate it.
-                        clustersUsedByTask.forEach(each -> each.stop(true));
+                        clustersUsedByTask.forEach(each -> stopCluster(each, true));
```

Comment:
`each` is a weird parameter name here. I think `cluster` would be a bit more clear.